models/sharky/solana/sharky_solana_events.sql

```diff
@@ -77,65 +83,118 @@ WITH sharky_txs AS (
         INNER JOIN filtered_txs
```

Comment:
Could we try switching the join order to
```
FROM filtered_txs
INNER JOIN sharky_txs
```
?

Reply:
This does the trick - see [sample PR](https://github.com/duneanalytics/spellbook/pull/3983)

Reply:
Excellent 🚀 Do you want to commit the changes yourself or should I do it?

Reply:
Feel free to commit the change yourself 🙌 thank you!